recipe/meta.yaml

```diff
@@ -24,11 +24,10 @@ requirements:
     - setuptools
     - numpy
     - cython
-    - freetype
+    - freetype  # [not win]
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - freetype
```

Comment:
@hmaarrfk How does this work?

Reply:
Freetype declares that it requires itself as a dependency if it is in the `run` section.

https://github.com/conda-forge/freetype-feedstock/blob/master/recipe/meta.yaml#L18

Reply:
FreeType _is_ a dependency if you aren't on Windows.

Reply:
I don't understand what `run_exports` does in that recipe.

Reply:
OK. I sort of understand now but it seems overly complicated.

Reply:
When a package has a `run_exports`, it tells conda that when it is used in the build process, it must be used in the included in the run section with the appropriate constraints.